paperqa/settings.py

```diff
@@ -302,9 +302,9 @@ def check_select(cls, v: str) -> str:
     def check_post(cls, v: str | None) -> str | None:
         if v is not None:
             # kind of a hack to get list of attributes in answer
-            from paperqa.types import Answer
+            from paperqa.types import PQASession
```

Comment:
Can we move this to the module level? Or will it cause a circular import

Reply:
yea - circular import is why this is here